code/modules/projectiles/projectile.dm

```diff
@@ -151,10 +151,10 @@
 				log_attack("<font color='red'>UNKNOWN/(no longer exists) shot [key_name(M)] with a [type]</font>")
 
 	if(A)
-		if(firer && istype(A, /obj/structure/stool/bed/chair/vehicle))//This is very sloppy but there's no way to get the firer after its passed to bullet_act, we'll just have to assume the admins will use their judgement
-			var/obj/structure/stool/bed/chair/vehicle/JC = A
-			if(JC.buckled_mob)
-				var/mob/BM = JC.buckled_mob
+		if(firer && istype(A, /obj/structure/bed/chair/vehicle))//This is very sloppy but there's no way to get the firer after its passed to bullet_act, we'll just have to assume the admins will use their judgement
+			var/obj/structure/bed/chair/vehicle/JC = A
+			if(JC.locked_atoms.len)
+				var/mob/BM = JC.locked_atoms[1]

```

Comment:
This is not the ideal method of doing this, the vehicle should have an explicit occupant var instead.